### PR TITLE
Display heart badge on loved comments

### LINKED ISSUE #655: Show creator-liked badge on comments
Some comments are liked by the uploader of the video and the comment has an icon next to it on YouTube and Invidious. It would be useful to add it to FreeTube too.

### PR DESCRIPTION
**Pull request by Synkky**
**Pull Request Type**
- Feature Implementation

**Related issue**
Should close #655

**Description**
This PR appends a badge to comments that were loved by the creator

**Screenshots (if appropriate)**
image
image
image

**Desktop:**
 - FreeTube version: development HEAD

### CODE REVIEW

File: src/renderer/components/watch-video-comments/watch-video-comments.css [hunk 1]
```diff
@@ -61,6 +61,43 @@
...
+.commentHeartBadgeImg {
+  position: absolute;
+  left: 0;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  -webkit-border-radius: 50%;
+}
+
+.commentHeartBadgeWhite {
```
Comment (PrestonN) [inline src/renderer/components/watch-video-comments/watch-video-comments.css:82]: Let's remove this and instead give `commentHeartBadgeRed` an outline.

File: src/renderer/components/watch-video-comments/watch-video-comments.css [hunk 2]
```diff
@@ -61,6 +61,43 @@
...
+  left: 9px;
+  bottom: 1px;
+  width: 11px;
+  height: 11px;
+  z-index: 1;
+}
+
+.commentHeartBadgeRed {
+  position: absolute;
+  color: red;
```
Comment (PrestonN) [inline src/renderer/components/watch-video-comments/watch-video-comments.css:93]: Please use the Material Design Red-500 as the color instead (#F44336).

**Review by PrestonN — CHANGES_REQUESTED** [review]
<3

Love it.  Looks really good!  Please make these small changes along with fixing your linting issues and I'm good to merge.

File: src/renderer/components/watch-video-comments/watch-video-comments.css (hunk 1 above)
Comment (Synkky) [reply src/renderer/components/watch-video-comments/watch-video-comments.css:82]: How do I give it an outline?
The only way I've done so is to use layers to place a regular white heart icon over the solid red heart icon.
Is there any other way?

Comment (PrestonN) [reply src/renderer/components/watch-video-comments/watch-video-comments.css:82]: I was hoping either `outline` or `text-stroke` would have worked just fine, but I wasn't having any luck getting this to work.  I'll consider this solution as fine then.

Comment (Synkky) [reply src/renderer/components/watch-video-comments/watch-video-comments.css:82]: If you are fine with me adding a dependency (@fortawesome/free-regular-svg-icons), I genuinely think it looks better

Comment (PrestonN) [reply src/renderer/components/watch-video-comments/watch-video-comments.css:82]: Yeah let's keep the icon we have for now.  If we decide to bring in the other icon pack as well for a future reason then I'd be open to changing this.

**Comment by Synkky** [comment]
That should be about it :)

**Review by PrestonN — APPROVED** [review]

**Comment by PrestonN** [comment]
Looks good to me.  Thank you for the contribution!

**Pull request merged**